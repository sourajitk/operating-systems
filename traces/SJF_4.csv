1,0,100
2,100,100
3,100,10
4,200,10
5,200,100
6,200,1000
7,300,10
8,300,10
9,400,10
10,400,10
11,400,10
12,500,10
13,500,10
14,500,10
15,600,10
16,700,10
17,700,10
18,700,10
19,700,10
20,700,10
