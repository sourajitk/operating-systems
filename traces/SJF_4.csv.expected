1,100
2,200
3,210
4,220
5,320
6,1340
7,330
8,340
9,1350
10,1360
11,1370
12,1380
13,1390
14,1400
15,1410
16,1420
17,1430
18,1440
19,1450
20,1460
